docs/dapp/tools/band.md

```diff
@@ -1,7 +1,7 @@
-# Integrating BAND oracle smart contract
+# Integrating BAND oracle
 
 This guide will explain how to query the Band Protocol reference data smart
-contract from another Solidity smart contract on Emerald.
+contract from another Solidity smart contract on an Oasis Paratime.
```

Comment:
Probably implied that it is an "EVM compatible Paratime" but perhaps we do need to emphasize that delineation in the tools section.